python/python/tests/utils/utils.py

```diff
@@ -79,15 +81,29 @@ async def check_if_server_version_lt(client: TRedisClient, min_version: str) ->
 
 
 def compare_maps(
-    map1: Optional[Union[Mapping[str, TResult], Dict[str, TResult]]],
-    map2: Optional[Union[Mapping[str, TResult], Dict[str, TResult]]],
+    map1: Optional[
+        Union[
+            Mapping[str, TResult],
+            Dict[str, TResult],
+            Mapping[bytes, TResult],
+            Dict[bytes, TResult],
+        ]
+    ],
+    map2: Optional[
+        Union[
+            Mapping[str, TResult],
+            Dict[str, TResult],
+            Mapping[bytes, TResult],
+            Dict[bytes, TResult],
+        ]
+    ],
 ) -> bool:
     """
     Compare two maps by converting them to JSON strings and checking for equality, including property order.
 
     Args:
-        map1 (Optional[Union[Mapping[str, TResult], Dict[str, TResult]]]): The first map to compare.
-        map2 (Optional[Union[Mapping[str, TResult], Dict[str, TResult]]]): The second map to compare.
+        map1 (Optional[Union[Mapping[Union[str, bytes], TResult], Dict[Union[str, bytes], TResult]]]): The first map to compare.
```

Comment:
Define new type `GlideString = Union[str, bytes]`. We have that in java client.

Reply:
It is not acceptable in Python to do this

Reply:
We do `TClusterResponse` same way, why not to do `GlideString`?

Reply:
The decision made by Bar and Ifarah is as of now, but might change in the future